src/kaleidoscope/hid.cpp

```diff
@@ -185,7 +358,9 @@ void releaseSystemControl(Key mappedKey) {
 }
 
 
-// Mouse events
+/***************
+ ***************     Mouse
+ ***************/
```

Comment:
This block is not related to the changes for this PR.

Reply:
Yes, it'll end up in another commit when I do the squash and final merge.